index.html

```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+    </section>
+    <section class="about">
+        <h1>About me</h1>
+        <p>
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If
+             you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a class="button" href="">Get My Resume</a>
+        <span class="horizontal-line"></span>
+        <ul class="about-list">
+            <li id="diamond">
+                Languages
+                <ul>
+                    <li><a href="">Javascript</a></li>
+                    <li><a href="">Ruby</a></li>
+                    <li><a href="">Html</a></li>
+                    <li><a href="">Css</a></li>
+                </ul>
```

Comment:
- [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of languages and not a link 😄.